components/mailer.rst

```diff
@@ -119,7 +119,10 @@ Load Balancing
 --------------
 
 Symfony's mailer supports `load balancing`_ via a technique called "round-robin"
-to distribute the mailing workload across multiple transports .
+to distribute the mailing workload across multiple transports. The only
+requirement is that your application must use the Messenger component
+integration to :ref:`send emails asynchronously <mailer-sending-emails-async>`,
+as explained in the next sections.
```

Comment:
I'm going to submit a patch to make it useful even if you are not using Messenger ( I forgot about this issue).